apps/sso/components/login-card/login-card.tsx

```diff
@@ -58,20 +58,20 @@ export function LoginCard() {
         height={100}
       />
       <H3>
-        Login to your{' '}
+        Log in to your{' '}
         <ColorSpanBold color={Colors2023.BLUE.STANDARD}>
           HackSC Account
         </ColorSpanBold>
       </H3>
       <StyledForm onSubmit={handleSubmit}>
-        <OneLineText
-          placeholder="enter your email"
+        <Input
+          placeholder="sample@email.edu"
```

Comment:
I think we can leave the placeholders as they were, even though it is different on the Figma. I think "enter your email" and "enter your password" are more descriptive